packages/connect-voting-disputable/tsconfig.json

```
@@ -0,0 +1,10 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "dist",
```

Just to align with the other packages ([it is also specified in the package.json](https://github.com/aragon/connect/pull/178/files#diff-70f05e0405597eaccd6b66bce74da065R11)):

```suggestion
    "outDir": "dist/esm",
```